src/Runner.Listener/BrokerMessageListener.cs

```diff
@@ -42,7 +42,7 @@ public override void Initialize(IHostContext hostContext)
             _brokerServer = HostContext.GetService<IBrokerServer>();
         }
 
-        public async Task<Boolean> CreateSessionAsync(CancellationToken token)
+        public async Task<int> CreateSessionAsync(CancellationToken token)
```

Comment:
At this layer, I wonder whether an enum would be more appropriate? I'm not sure it's appropriate for this layer to understand application return codes.